Sources/NIOHTTP1/HTTPTypes.swift

```diff
@@ -19,11 +19,52 @@ let headerSeparator: StaticString = ": "
 
 /// A representation of the request line and header fields of a HTTP request.
 public struct HTTPRequestHead: Equatable {
+    private class _Storage {
```

Comment:
I guess it not really makes a difference but use `final` (for all internal classes) ?